test/kubemark/aks/util.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script contains the helper functions that each provider hosting
+# Kubermark must implement to use test/kubemark/start-kubemark.sh and
+# test/
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/../../..
+
+source "${KUBE_ROOT}/test/kubemark/common/util.sh"
+
+# This function should authenticate docker to be able to read/write to
+# the right container registry (needed for pushing kubemark image).
+function authenticate-docker {
+    echo "Configuring registry authentication" 1>&2
```

Comment:
nothing is done here?

Reply:
https://github.com/kubernetes/kubernetes/blob/2bacdf8047482da89b14e679c65ab61e0486553b/test/kubemark/start-kubemark.sh#L49

needs to exist so `errexit` doesn't force failure.